### ISSUE TITLE
Polish the Android A-S Metrics Summary Dashboard

### ISSUE THREAD

**Issue by emtwo** [mozilla/ping-centre#105]
https://sql.telemetry.mozilla.org/dashboard/android-activity-stream-proof-of-concept-

It should split by release channel

**Comment by mcomella** [comment]
Another issue I found: for the "Interactions by Week" graph, we have two different events that have `source_type = "pocket"`:
1. clicks on specific Pocket stories (i.e. row items)
1. clicks on the "More" button to open Pocket's trending page to show more articles.

It'd be great to distinguish them in the graph. They're distinguishable by the extras, specifically "More" links include the extra `item = "link_more"` and specific Pocket stories do not have this extra. See [the docs for more details](https://firefox-source-docs.mozilla.org/mobile/android/fennec/activitystreamtelemetry.html#top-stories-pocket-interactions).

**Comment by mcomella** [comment]
FYI: Maria requested two things for release:
1. A dashboard for release data
1. Interactions for opening new tab, add bookmark, etc., like [this iOS query](https://sql.telemetry.mozilla.org/queries/5799#11788)

I'm currently working on this. I forked the dashboard to ["Android: Activity Stream Metrics Summary (release)"](https://sql.telemetry.mozilla.org/dashboard/android-activity-stream-metrics-summary-release-), which should show release data once 57 is released and I'm currently working on adding 2).

My best efforts will be hacky so I think we should consolidate my changes into the changes that should have been made for this bug. :)

**Comment by mcomella** [comment]
I completed my work on the dashboards including adding context menu interactions. A few notes:
- I changed the "Top sites/Pocket/highlights" query so that it would not include opening context menus or selecting an item in the context menu. It could be misleading that the two dashboards don't display the same data. I documented this divergence in the description of the query itself.
- We don't currently display how often a context menu is opened (pending feedback from Maria)
- Pocket clicks only make sense when pocket is enabled and we only have pocket enabled in ~3 locales – ideally, we'd take that into account (Maria said I don't need to fix for now). To a lesser degree, this also affects highlights, which could be disabled
- (To repeat my previous comment) Pocket clicks include clicks on the Pocket "More" button (Maria said I don't need to fix for now)
